tdm-be/bin/swagger.ts

```diff
@@ -1,5 +1,8 @@
-const swaggerJsdoc = require('swagger-jsdoc');
-const fs = require('fs');
+import swaggerJsdoc from 'swagger-jsdoc';
+import { writeFileSync } from 'fs';
+
+// Replace this with http://localhost:3000 in dev mode and run `make update-front-api`
+const HOST = '';
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏Should come from the environment

So that there would be no need to modify source code in dev mode.

Reply:
Yes, but we still need to run `make update-front-api` who run two commands, one from the backend and one from the frontend.